### DISCUSSION TITLE
Image input field & background-image style

### DISCUSSION THREAD

**Discussion by esisl** [pimcore/pimcore#7715] [Q&A / Support] [answered]
I can insert an editable image into a page like this:

> < ? = $this->image("myImage", ["thumbnail"=>"square500"]); ?>

But I need the image to load here:
> < div style="background-image: url( myImageFileName.png );">

Is this possible in principle?
If so, how?

**Comment by dvesh3** [comment, marked as answer]
@esisl please look at custom image tag example https://pimcore.com/docs/pimcore/current/Development_Documentation/Documents/Editables/Image.html#page_Other-Advanced-Examples

**Comment by esisl** [comment]
Thanks.
This is what you need!
